install.sh

```diff
@@ -270,6 +319,36 @@ libsecp256k1_install()
     popd
 }
 
+clightning_build ()
+{
+    ./configure \
```

Comment:
```suggestion
    PG_CONFIG="" ./configure \
```

I don't have PostgreSQL installed on machines I'm currently trying to test this, but randomly noticed looking at c-lightning's configure script (for different reasons), this should disable building PostgreSQL support for c-lightning inside jmvenv, which is not necessary there, thus could little bit improve build speed. But this is a minor detail, of course.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1000/commits/97cf02c50fc08661e8f47df480a40212e8fc01de